std/traits.d

```diff
@@ -8812,6 +8812,20 @@ template getSymbolsByUDA(alias symbol, alias attribute)
     static assert(is(getSymbolsByUDA!(X, X) == AliasSeq!()));
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=23776
+@safe pure nothrow @nogc unittest
+{
+    struct T
+    {
+        struct Tag {}
+        @Tag struct MyStructA {}
+        @Tag struct MyStructB {}
+        @Tag struct MyStructC {}
+    }
+    alias components = getSymbolsByUDA!(T, T.Tag);
+    static assert(components.length > 0); 
```

Comment:
```suggestion
    static assert(components.length > 0);
```

Reply:
don't shoot, not my fault. @quickfur had a trailing white space in his bug report, I just pasted it. :-"